lib/status.js

```diff
@@ -28,10 +28,7 @@ exports.checkStatus = async function(options = {}) {
     } else {
       const error = new Error(
         `Unsupported credentialStatus type "${credentialStatus.type}".`);
-      // throw error;
-      console.error(error);
-      // FIXME: Throw error instead of returning verified
-      return {verified: true};
```

Comment:
This FIXME was added to ensure we comply with vendors in the VC HTTP API Test Suite that set their `credentialStatus` endpoint to an erroneous endpoint. This version of code must be tested agains the VC HTTP API before merge.

Reply:
@gannan08 Can you point me to those VC HTTP API tests?

Reply:
Yes - https://github.com/w3c-ccg/vc-http-api/tree/main/packages/vc-http-api-test-server

Reply:
The status on this is that 9 tests are failing when verifying presentation and I get this error https://gist.github.com/JSAssassin/8003894b5523721a48dfca9f2261cf19

Reply:
Update: Currently when using this branch on veres-verifier there are 5 failing tests on vc-http-api test suite. The current solution that makes those tests pass is loading the document locally via the documentLoader which is in draft PR here https://github.com/digitalbazaar/bedrock-vc-verifier/pull/16.